### PR TITLE
Follow-up edits to PR#2111

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2111

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf @tpoitras PTAL. Thanks!

File: admin_guide/scoped_tokens.adoc [hunk 1]
```diff
@@ -11,39 +11,43 @@
 toc::[]
 
 == Overview
-A user may wish to give another entity the power to act as him, but only in a limited way.
-For example, a project admin may wish to delegate the power to create pods.  One way to do 
-this is to create a scoped token.
-
-A scoped token, is a token that identifies as a given user, but is limited to certain actions 
-by its scope.  Right now, only a cluster-admin can create scoped tokens.
+A user may wish to give another entity the power to act as him or her, but only
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: s/wish/want

Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: I'd avoid this him/her gender dichotomy, a good rule in general for technical writing.

Suggest rephrase with "they":  A user may want to give another entity the same power to act that they have, but only in a limited way.

File: admin_guide/scoped_tokens.adoc [hunk 2]
```diff
@@ -11,39 +11,43 @@
...
+in a limited way. For example, a project administrator may want to delegate the
+power to create pods. One way to do  this is to create a scoped token.
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: double space

File: admin_guide/scoped_tokens.adoc [hunk 3]
```diff
@@ -11,39 +11,43 @@
...
 
+A scoped token is a token that identifies as a given user, but is limited to
+certain actions  by its scope.  ight now, only a *cluster-admin* can create
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: s/ight/Right

File: admin_guide/scoped_tokens.adoc [hunk 4]
```diff
@@ -11,39 +11,43 @@
...
-== Scopes
-=== User Scopes
-User scopes are focused on getting information about a given user.  They are intent-based, so
-the rules are automatically created for you.
-
- 1. `user:info` - Allows access to information the user: name, groups, etc.
- 2. `user:check-access` - Allows access to self-localsubjectaccessreviews and self-subjectaccessreviews.
+Scopes are evaluated by converting the set of scopes for a token into a set of
+*PolicyRules*. Then, the request is matched against those rules. The request
+attributes must match at least  one of the scope rules to be passed to the
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: double space

File: admin_guide/scoped_tokens.adoc [hunk 5]
```diff
@@ -11,39 +11,43 @@
...
+"normal" authorizer for further authorization checks.
+
+[[admin-guide-scoped-tokens-user-scopes]]
+== User Scopes
+User scopes are focused on getting information about a given user. They are
+intent-based, so the rules are automatically created for you.
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: I'd end this with a colon:
s/you./you:

File: admin_guide/scoped_tokens.adoc [hunk 6]
```diff
@@ -11,39 +11,43 @@
...
+
+* `user:info` - Allows access to information the user: name, groups, etc.
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: - Allows access to information ABOUT the user
- avoid latin abbreviations, in this case "etc."
- I'd also do away with that mid-sentence colon.

Suggest: ... Allows access to information about the user, such as the user name and groups.

File: admin_guide/scoped_tokens.adoc [hunk 7]
```diff
@@ -11,39 +11,43 @@
...
+* `user:check-access` - Allows access to *self-localsubjectaccessreviews* and *self-subjectaccessreviews*.
     Those are the variants where you pass an empty user and groups in your request object.
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: should "variants" be "variables"? I don't know, I'm just asking. I don't think I remember seeing "variants" in our docs aside from choosing which version of OpenShift to install.

Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: s/Those/These 
?
Maybe?

File: admin_guide/scoped_tokens.adoc [hunk 8]
```diff
@@ -11,39 +11,43 @@
...
-=== Role Scope
-The role scope is about having the same level of access as a given role filtered by namespace.
+[[admin-guide-scoped-tokens-role-scope]]
+== Role Scope
+The role scope allows you to have the same level of access as a given role
+filtered by namespace.
 
- 1. `role:<cluster-role name>:<namespace or * for all>` - Limits the scope to the rules specified
+* `role:<cluster-role name>:<namespace or * for all>` - Limits the scope to the rules specified
     by the cluster-role, but only in the namespace specified.
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: minor nit, you can ignore this one if you disagree, but I'd reverse the last two words so it's: ... in the specified namespace. Just a flow thing.

File: admin_guide/scoped_tokens.adoc [hunk 9]
```diff
@@ -11,39 +11,43 @@
...
-
-    Caveat: This prevents escalating access.  Even if the role allows access to resources like
-    secrets, rolebindings, and roles, this scope will *deny* access to those resources.  This helps
-    prevent unexpected escalations.  Many people do not think of a role like `edit` as being an escalating
-    role, but with access to a `secret`, it is.
-
- 1.  `role:<cluster-role name>:<namespace or * for all>:!` - Note the bang.  This is like #1, except that it
++
+Caveat: This prevents escalating access. Even if the role allows access to
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: I'd format this "caveat" as a NOTE admonition.

File: admin_guide/scoped_tokens.adoc [hunk 10]
```diff
@@ -11,39 +11,43 @@
...
+resources like secrets, rolebindings, and roles, this scope will *deny* access
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: why is "deny" marked up?

File: admin_guide/scoped_tokens.adoc [hunk 11]
```diff
@@ -11,39 +11,43 @@
...
+to those resources. This helps prevent unexpected escalations. Many people do
+not think of a role like *edit* as being an escalating role, but with access to
+a secret it is.
+
+* `role:<cluster-role name>:<namespace or * for all>:!` - Note the bang. This is similar to above, except that it
      *does* allow escalating access.
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: The word "does" appears to have needless markup here as well

File: admin_guide/scoped_tokens.adoc [hunk 12]
```diff
@@ -11,39 +11,43 @@
...
-
- 1.  `role:<cluster-role name>:<namespace or * for all>:!` - Note the bang.  This is like #1, except that it
++
+Caveat: This prevents escalating access. Even if the role allows access to
+resources like secrets, rolebindings, and roles, this scope will *deny* access
+to those resources. This helps prevent unexpected escalations. Many people do
+not think of a role like *edit* as being an escalating role, but with access to
+a secret it is.
+
+* `role:<cluster-role name>:<namespace or * for all>:!` - Note the bang. This is similar to above, except that it
```
Comment (tpoitras) [inline admin_guide/scoped_tokens.adoc]: Avoid starting sentences with "Note that..." or "Note the...". Suggest rephrase: This is similar to the example above, except that including the bang causes this scope to allow escalating access.

**Comment by tpoitras** [comment]
@ahardin-rh 
I made some comments. 
:zap: :palm_tree: :ice_cream:

**Comment by bfallonf** [comment]
@ahardin-rh Pretty much what Tim said. Otherwise, looks fine!

**Comment by ahardin-rh** [comment]
@bfallonf @tpoitras Thanks! All fixed :wrench:

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/scoped_tokens.adoc#admin-guide-scoped-tokens[Scoped Tokens]
|New topic discussing scoped tokens, which are tokens that identify given users, but are limited to certain actions by its scope.
%

**Pull request merged**